consensus/pbft_log.go

```diff
@@ -0,0 +1,76 @@
+package consensus
+
+import (
+	mapset "github.com/deckarep/golang-set"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/core/types"
+)
+
+// PbftLog represents the log stored by a node during PBFT process
```

Comment:
why use a new log package?

Reply:
shouldn't it be in utils/ module?

Reply:
sorry for naming confusing. it's not the log for debugging, it's the storage/log for consensus messages.